.github/workflows/docker-build.yaml

```diff
@@ -45,13 +45,21 @@ jobs:
         # Setup docker build to use your Namespace workspace builder
         name: Set up Namespace Buildx
         uses: namespacelabs/nscloud-setup-buildx-action@v0
+      - name: Build runwasi as static muls for amd64 and arm64
+        run: |
+          git clone --branch containerd-shim-wasm/v0.3.0 --depth 1 https://github.com/containerd/runwasi.git &&
+          cd runwasi &&
+          docker build --platform=linux/arm64,linux/amd64 --build-arg BASE_IMAGE=alpine --build-arg CRATE="containerd-shim-wasmtime,containerd-shim-wasmedge,containerd-shim-wasmer" -t nscr.io/a8fcp47vcfori/build-runwasi --push . &&
```

Comment:
@jprendes, do you know why `wasmedge`build is broken via `docker build --platform=linux/arm64,linux/amd64 --build-arg BASE_IMAGE=alpine --build-arg CRATE="containerd-shim-wasmedge"` I get an error that it would need `pthreads`:
```log
#32 93.43 error[E0425]: cannot find function `pthread_sigqueue` in crate `libc`
#32 93.43     --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/wasmedge-sys-0.17.1/src/executor.rs:38:19
#32 93.43      |
#32 93.43 38   |             libc::pthread_sigqueue(dist_pthread, sig, si_value);
#32 93.43      |                   ^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `pthread_sigmask`
#32 93.43      |
#32 93.43     ::: /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/libc-0.2.147/src/unix/linux_like/linux/mod.rs:4623:5
#32 93.43      |
#32 93.43 4623 |     pub fn pthread_sigmask(how: ::c_int, set: *const sigset_t, oldset: *mut sigset_t) -> ::c_int;
#32 93.43      |     -------------------------------------------------------------------------------------------- similarly named function `pthread_sigmask` defined here
```

Reply:
Due to a change upstream: https://github.com/WasmEdge/wasmedge-rust-sdk/issues/70

Reply:
Opened a PR: https://github.com/WasmEdge/wasmedge-rust-sdk/pull/71